component/main.jsonnet

```diff
@@ -1,10 +1,19 @@
 // main template for openshift4-slos
+local com = import 'lib/commodore.libjsonnet';
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.openshift4_slos;
 
 // Define outputs below
-{
-}
+local mergeSpec = function(name, spec)
+  local slothRendered = std.parseJson(kap.yaml_load('%s/sloth-output/%s.yaml' % [ inv.parameters._base_directory, name ]));
```

Comment:
This would probably work without the base_directory prefix, but it doesn't hurt to have it either.